docs/deployments.adoc

```diff
@@ -0,0 +1,185 @@
+
+|Keep Stake
+|link:https://goerli.etherscan.io/address/0x10DE37cF84202A20cae61069C617B3Aa874aF8b4#code[0x10DE37cF84202A20cae61069C617B3Aa874aF8b4]
+
+|Token Staking 
+|link:https://etherscan.io/address/0x01b67b1194c75264d06f808a921228a95c765dd7#writeProxyContract[0x01b67b1194c75264d06f808a921228a95c765dd7]
+
+|RandomBeacon
+|link:https://etherscan.io/address/0x5499f54b4A1CB4816eefCf78962040461be3D80b[0x5499f54b4A1CB4816eefCf78962040461be3D80b]
+
+|WalletRegistry
+|link:https://etherscan.io/address/0xfbae130e06bbc8ca198861beecae6e2b830398fb[0xfbae130e06bbc8ca198861beecae6e2b830398fb]
+
+|Bridge
+|TBD
+
+|===
+
+== Görli Testnet
+
+=== Packages
+
+The most recent packages deployed to Görli are published to the npm Registry
+and tagged with _goerli_.
+
+Currently they are:
+```
+@keep-network/keep-core@1.8.1-goerli.0
+@keep-network/keep-ecdsa@1.9.0-goerli.0
+@keep-network/tbtc@1.1.2-goerli.0
+@keep-network/coverage-pools@1.1.0-goerli.0
+@threshold-network/solidity-contracts@1.3.0-goerli.0
+@keep-network/random-beacon@2.1.0-goerli.1
+@keep-network/ecdsa@2.1.0-goerli.1
+@keep-network/tbtc-v2@0.1.1-goerli.6
+```
+
+[#goerli-contracts]
+=== Contracts addresses
+
+[%header,cols="1m,1m"]
+|===
+|Contract
+|Address
+
+|Vending Machine (NU<>T)
+|link:https://goerli.etherscan.io/address/0x8D4Ccc04D17d055b88fbaB8634C7F0611d8Aa946#code[0x8D4Ccc04D17d055b88fbaB8634C7F0611d8Aa946]
+
+|Vending Machine (KEEP<>T)
+|link:https://goerli.etherscan.io/address/0x8d1EDfee0510537E1081a28Fa4A94769Cfa3969a#code[0x8d1EDfee0510537E1081a28Fa4A94769Cfa3969a]
+
+|T token
+|link:https://goerli.etherscan.io/address/0x3f16380656cAE45D3f80D8833682d2b606eD094A#code[0x3f16380656cAE45D3f80D8833682d2b606eD094A]
+
+|NU token
+|link:https://goerli.etherscan.io/address/0x26bDc8Cb2D4F55Af3B43568069b65B435bCdc32c#code[0x26bDc8Cb2D4F55Af3B43568069b65B435bCdc32c]
+
+|KEEP token
+|link:https://goerli.etherscan.io/address/0x22647FfAe391540d584599818CA22fdF18890753#code[0x22647FfAe391540d584599818CA22fdF18890753]
+
+|NU Staking Escrow (stub)
+|link:https://goerli.etherscan.io/address/0xd696d5a9b083959587F30e487038529a876b08C2#code[0xd696d5a9b083959587F30e487038529a876b08C2]
+
+|Keep Stake
+|link:https://goerli.etherscan.io/address/0xC76796B031232976cb17521342B0d12283588E6F#code[0xC76796B031232976cb17521342B0d12283588E6F]
+
+|Token Staking
+|link:https://goerli.etherscan.io/address/0x1da5d88C26EA4f87b5e09C3452eE2384Ee20DC75#code[0x1da5d88C26EA4f87b5e09C3452eE2384Ee20DC75]
+
+|RandomBeacon
+|link:https://goerli.etherscan.io/address/0xaFfCD4734eEa140Ba5666Bf60541CCAFfa74F4be#code[0xaFfCD4734eEa140Ba5666Bf60541CCAFfa74F4be]
+
+|WalletRegistry
+|link:https://goerli.etherscan.io/address/0x82BE0F8C8d43fAC584B03f4b782370E202A34527#writeProxyContract[0x82BE0F8C8d43fAC584B03f4b782370E202A34527]
+
+|Bridge
+|link:https://goerli.etherscan.io/address/0x52E861c35240d722e9EF7B605dC72C71ff8fc185#code[0x52E861c35240d722e9EF7B605dC72C71ff8fc185]
+
+|===
+
+== Ropsten Testnet
```

Comment:
I think we can get rid of ropsten.

Reply:
Agree - I've included only Goerli testnet in the Threshold docs.